packages/frontend/src/pages/MetricFlow.tsx

```diff
@@ -189,39 +187,64 @@ const MetricFlowPage = () => {
                                 </Badge>
                             </Tooltip>
                         </Flex>
-                        <Flex gap="xs" align="center">
-                            <Tooltip label={'Refetch fields'}>
-                                <ActionIcon
-                                    size="sm"
-                                    variant="outline"
-                                    color="blue"
-                                    loading={metricFlowFieldsQuery.isFetching}
+                        <Button.Group>
+                            <Tooltip
+                                label={'Run query'}
+                                withinPortal
+                                position="bottom"
+                            >
+                                <Button
+                                    size="xs"
+                                    variant="default"
+                                    disabled={
+                                        metricFlowQueryResultsQuery.isLoading
+                                    }
+                                    onClick={() =>
+                                        metricFlowQueryResultsQuery.refetch()
+                                    }
+                                >
+                                    <IconPlayerPlay size={12} color="blue" />
```

Comment:
you can use <MantineIcon icon={IconPlayerPlay} and then you'll get mantine like theming options like `size='sm'`